cmd/tether/attach.go

```diff
@@ -328,3 +364,11 @@ func (t *attachServerSSH) channelMux(in <-chan *ssh.Request, process *os.Process
 
 	detach()
 }
+
```

Comment:
The correct solution is to have this in Go standard libaries and exposed as part of the syscall libraries. The docker one is for Windows.